### DISCUSSION TITLE
Plans for supporting OpenSSL 3.0?

### DISCUSSION THREAD

**Discussion by HKGx** [prisma/prisma#11345] [Q&A] [answered]
OpenSSL 3.0 has been released in September and probably is pretty much backwards compatible with OpenSSL 1.1.1. Are there any plans for supporting it?
https://www.openssl.org/blog/blog/2021/09/07/OpenSSL3.Final/

**Comment by pantharshit00** [comment, marked as answer]
As far as I remember openssl is forwards compatible so you should not have any trouble running Prisma with OpenSSL 3.0 but I just opened https://github.com/prisma/prisma/issues/11356 to properly track this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
